docs/director/fetch-requests.md

```diff
@@ -0,0 +1,20 @@
+# Fetch Requests
```

Comment:
Please also include this document in the general doc with listing all documents.